2-Authorization-I/2-call-arm/README.md

```diff
@@ -0,0 +1,520 @@
+   1. Select the **Add a permission** button and then:
+
+   1. Ensure that the **Microsoft APIs** tab is selected.
+   1. In the list of APIs, select the API `Windows Azure Service Management API`.
+      * Since this app signs-in users, we will now proceed to select **delegated permissions**, which is is requested by apps when signing-in users.
+           1. In the **Delegated permissions** section, select the **user_impersonation** in the list. Use the search box if necessary.
+   1. Select the **Add permissions** button at the bottom.
+   1. Select the **Add a permission** button and then:
+
+   1. Ensure that the **Microsoft APIs** tab is selected.
+   1. In the list of APIs, select the API `Azure Storage`.
+      * Since this app signs-in users, we will now proceed to select **delegated permissions**, which is is requested by apps when signing-in users.
+           1. In the **Delegated permissions** section, select the **user_impersonation** in the list. Use the search box if necessary.
+   1. Select the **Add permissions** button at the bottom.
+
+##### Assign Azure role-based access control (Azure RBAC)
+
+1. Ensure that [An Azure Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal) was created, and if not please create one.
+1. Assign the role `Storage Blob Data Contributor` to your user or group to have read and write access to your blob storage. Please see [Assign Azure roles using the Azure portal](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=current).
+
+##### Configure the spa app (ms-identity-react-c2s2) to use your app registration
+
+Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
+
+> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `SPA\src\authConfig.js` file.
+1. Find the key `Enter_the_Application_Id_Here` and replace the existing value with the application ID (clientId) of `ms-identity-react-c2s2` app copied from the Azure portal.
+1. Find the key `Enter_the_Tenant_Info_Here` and replace the existing value with your Azure AD tenant/directory ID.
+1. find the key `Enter_Storage_Account_Name` and replace the existing value with your storage account name.
+
+### Step 4: Running the sample
+
+From your shell or command line, execute the following commands:
+
+```console
+    cd 2-Authorization-I\2-call-arm\SPA
+    npm start
+```
+
+## Explore the sample
+
+1. Open your browser and navigate to `http://localhost:3000`.
+1. Select the **Sign In** button on the top right corner.
+1. Select the **Tenants** button on the navigation bar. This will make a call to the Azure Resource Manager API.
+![Screenshot](./ReadmeFiles/screenshotOne.png)
+1. Select the **Storage** button on the navigation bar. After that use the form upload a file to Azure Storage.
```

Comment:
Nice to have: storage acccount page shows if I have any files/how many files I have in my container